AstSemantics.md

```diff
@@ -179,6 +179,9 @@ Thus, it is recommend that WebAssembly producers align frequently-used data to
 permit the use of natural alignment access, and use loads and stores with the
 greatest alignment values practical, while always avoiding misaligned accesses.
 
+### Order of evaluation
+
+Operand evaluation order is deterministic. All nodes other than control flow constructs need to evaluate their child nodes in the order they appear in the AST.
```

Comment:
This wording isn't very clear; e.g. there are a number of different "orderings" for tree walks.

Suggest wording the first sentence as "When evaluating an operation, operands are always evaluated in 'left-to-right' order" and then given an example, such as int32_add.